cellarium/ml/core/module.py

```diff
@@ -59,33 +52,27 @@ def __init__(
         optim_kwargs: dict[str, Any] | None = None,
         scheduler_fn: type[torch.optim.lr_scheduler.LRScheduler] | None = None,
         scheduler_kwargs: dict[str, Any] | None = None,
-        default_lr: float = 1e-3,
-        config: dict[str, Any] | Namespace | None = None,
     ) -> None:
         super().__init__()
+
+        _transforms, _model = transforms, model
+
+        transforms = [uninitialize_object(transform) for transform in _transforms] if _transforms is not None else None
```

Comment:
Can you explain this to me?

Reply:
Perhaps also write two lines of comment about why you're uninitializing and reinitializing?

Reply:
Added a comment above explaining the logic behind un-initializing and initializing.

Reply:
model and transforms are not attributes of self (yet)-- does uninitializing them have any effect on the behavior of `self.save_hyperparameters`?